docs/release.sh

```diff
@@ -116,17 +84,20 @@ invalidate_cache() {
 
 	aws configure set preview.cloudfront true
 
-	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
+	# Get all the files
+	# not .md~ files
+	# not files w spaces bc they mess up the array
```

Comment:
you should also remove this comment then :)

Also did you intentionally remove a backslash from the sed ? used to be `s/site\///g' and is noe `s/site//g`.